src/patternfly/components/CalendarMonth/calendar-month.scss

```diff
@@ -0,0 +1,214 @@
+  font-weight: var(--pf-c-calendar-month__day--FontWeight);
+  text-align: center;
+}
+
+.pf-c-calendar-month__dates-row:first-child {
+  --pf-c-calendar-month__dates-cell--PaddingTop: var(--pf-c-calendar-month__dates-row--first-child__dates-cell--PaddingTop);
+}
+
+.pf-c-calendar-month__dates-cell {
+  position: relative;
+  padding: var(--pf-c-calendar-month__dates-cell--PaddingTop) var(--pf-c-calendar-month__dates-cell--PaddingRight) var(--pf-c-calendar-month__dates-cell--PaddingBottom) var(--pf-c-calendar-month__dates-cell--PaddingLeft);
+  text-align: center;
+
+  &::before {
+    position: absolute;
+    top: var(--pf-c-calendar-month__dates-cell--before--Top);
+    right: var(--pf-c-calendar-month__dates-cell--before--Right);
+    bottom: var(--pf-c-calendar-month__dates-cell--before--Bottom);
+    left: var(--pf-c-calendar-month__dates-cell--before--Left);
+    content: "";
+    background-color: var(--pf-c-calendar-month__dates-cell--before--BackgroundColor);
+  }
+
+  &.pf-m-current {
+    --pf-c-calendar-month__date--BackgroundColor: var(--pf-c-calendar-month__dates-cell--m-current__date--BackgroundColor);
+  }
+
+  &.pf-m-in-range {
+    --pf-c-calendar-month__dates-cell--before--BackgroundColor: var(--pf-c-calendar-month__dates-cell--m-in-range--before--BackgroundColor);
+
+    &:not(.pf-m-start-range):not(.pf-m-end-range) {
+      --pf-c-calendar-month__date--hover--after--BorderColor: transparent;
+      --pf-c-calendar-month__date--focus--after--BorderColor: transparent;
+    }
+  }
+
+  &.pf-m-start-range {
+    --pf-c-calendar-month__dates-cell--before--Left: var(--pf-c-calendar-month__dates-cell--m-in-range--m-start-range--before--Left);
+  }
+
+  &.pf-m-end-range {
+    --pf-c-calendar-month__dates-cell--before--Right: var(--pf-c-calendar-month__dates-cell--m-in-range--m-end-range--before--Right);
+  }
+
+  &.pf-m-selected {
+    --pf-c-calendar-month__date--BackgroundColor: var(--pf-c-calendar-month__dates-cell--m-selected__date--BackgroundColor);
+    --pf-c-calendar-month__date--Color: var(--pf-c-calendar-month__dates-cell--m-selected__date--Color);
+  }
+
+  &.pf-m-adjacent-month {
+    --pf-c-calendar-month__date--Color: var(--pf-c-calendar-month__dates-cell--m-adjacent-month__date--Color);
+  }
+}
+
+.pf-c-calendar-month__date {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: var(--pf-c-calendar-month__date--Width);
+  height: var(--pf-c-calendar-month__date--Height);
+  line-height: 1;
+  color: var(--pf-c-calendar-month__date--Color);
+  background-color: var(--pf-c-calendar-month__date--BackgroundColor);
+  border: 0;
+
+  &::after {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border: var(--pf-c-calendar-month__date--after--BorderWidth) solid var(--pf-c-calendar-month__date--after--BorderColor);
+  }
+
+  &,
+  &::after {
+    border-radius: var(--pf-c-calendar-month__date--BorderRadius);
+  }
```

Comment:
I think the default focus ring is visually confusing:

![Screen Shot 2020-11-03 at 4 09 35 PM](https://user-images.githubusercontent.com/5385435/98260666-96569200-1f51-11eb-921e-91f7d55d6b49.png)

Perhaps we should disable default `:focus` with `outline: none;` since focus styles are explicitly applied below in the CSS.

@mcarrano wdyt?

Reply:
@mattnolting I agree. Also @jessiehuff wdyt about that?

Reply:
I'm good with this suggestion. The general recommendation is that if you're going to remove the native focus styles, that you must replace them with custom CSS that indicate focus. Jenn's told me that at the beginning of PF4, we took the stance of relying on browser styles for indicating focus when possible, and providing custom CSS when browser styles weren't available (i.e. the Switch component). But there are probably situations where native browser styles are present but not ideal for some reason, and this is probably a more gray, subjective area. I think in cases where we're doing a really custom component (i.e. not a native html element) that chances are there are more reasons in favor of using custom css for focus than native browser styles.